FaceDetection/CMakeLists.txt

```diff
@@ -1,17 +1,23 @@
-cmake_minimum_required(VERSION 2.8.4)
+cmake_minimum_required(VERSION 3.5.1)
 
 project(seeta_facedet_lib)
 
 # Build options
 option(BUILD_EXAMPLES  "Set to ON to build examples"  ON)
 option(USE_OPENMP      "Set to ON to build use openmp"  ON)
+option(USE_SSE         "Set to ON to build use SSE"  ON)
 
 # Use C++11
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 message(STATUS "C++11 support has been enabled by default.")
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.1")
+# Use SSE
+if (USE_SSE)
+    add_definitions(-DUSE_SSE)
+    message(STATUS "Use SSE")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -msse4.1")
+endif()
 
 # Use OpenMP
 if (USE_OPENMP)
```

Comment:
It seems that `OpenMP_FOUND` on line 25 should be `OPENMP_FOUND`. The former works fine for CMake 3.5 or newer, while the latter can also be used with CMake 3.1.